common/test/acceptance/pages/studio/settings_advanced.py

```diff
@@ -34,6 +37,30 @@ def _is_finished_loading():
         EmptyPromise(_is_finished_loading, 'Finished rendering the advanced policy items.').fulfill()
         return self.q(css='body.advanced').present
 
+    @property
+    def key_names(self):
+        """
+            Returns a list of key names of all settings.
+        """
+        return self.q(css=KEY_CSS).text
+
+    @property
+    def deprecated_settings_button_text(self):
+        """
+            Returns text for deprecated settings button
+        """
+        return self.q(css=DEPRECATED_SETTINGS_BUTTON_SELECTOR).text[0]
+
+    def click_deprecated_settings_button(self):
+        """
+            Show deprecated Settings
+        """
+        self.q(css=DEPRECATED_SETTINGS_BUTTON_SELECTOR).click()
+        if self.deprecated_settings_button_text == 'Show Deprecated Settings':
+            self.wait_for_element_presence(DEPRECATED_SETTINGS_SELECTOR, 'Deprecated Settings are not present')
```

Comment:
change message to "Deprecated Settings are present, see my comments on other branch